src/couch_stream.erl

```diff
@@ -51,7 +51,7 @@ open(Fd) ->
     open(Fd, []).
 
 open(Fd, Options) ->
-    gen_server:start_link(couch_stream, {Fd, self(), Options}, []).
+    gen_server:start_link(couch_stream, {Fd, {self(), erlang:get(io_priority)}, Options}, []).
```

Comment:
the init function would crash either way, so I think it's clearer if each required argument is expressed naturally.